### DISCUSSION TITLE
Welcome to Scribbler Community!

### DISCUSSION THREAD

**Discussion by gopi-suvanam** [gopi-suvanam/scribbler#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by purplemonk** [comment]
Will scribbler work on my mobile? I have only 10-20 mins to learn coding or rather programming. What can I expect from the Master Class

**Comment by purplemonk** [comment]
How do I do spectral analysis of an audio?

**Comment by purplemonk** [comment]
How do I use recursion in javascript? Could you provide a Recursion.jsnb? Is there a repository of .jsnb that can be using in Sribbler.live

**Comment by purplemonk** [comment]
Do we have a discord server or telegram or whatsapp group?

**Reply by gopi-suvanam** [reply]
Github discussions can be a good community platform. But you can join our discord server for more active participation: https://discord.gg/AzqpWh66
